server/src/controllers/job_applications/getAllJobApplications.js

```diff
@@ -0,0 +1,71 @@
+const Sequelize = require("sequelize");
+const logger = require("pino")();
+const JobApplications = require("../../models/job_applications.model");
+
+const getAllJobApplicationsRequestHandler = async (req, res) => {
+  try {
+    // ! Return all applications sorted in descending order by application date (newest first)
+    // ! There is probably a sequelize function that can do this for me.
+    const query = {};
+    const order = [["applied_date", "DESC"]];
```

Comment:
This PR removes the applied_date field from the model: https://github.com/uoftblueprint/tces/pull/127, please use the createdAt field instead